hearts/static/room_socket_client.js

```diff
@@ -15,15 +15,18 @@ function RoomSocketClient(apiUrl) {
     var data = {
       cards: cards
     };
-    var result = this.socket.emit('pass_cards', data);
-    return result.status;
+    this.socket.emit('pass_cards', data, function(result){
+      console.log('received confirmation', result);
```

Comment:
If it's a failure, do you want to display the message? If so, this will be annoying, because you'll need to add a function argument to this method (`passCards`) that stores a callback for displaying the message. Then inside *this* callback you have to invoke *that* callback with the returned result.

This is the beginning of we call "callback hell"

Reply:
For now, I just want to log the message instead of displaying it.  
I will need to display messages when a card is played, though.  However, I don't want to use a callback to display the message either.